src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -21,12 +21,12 @@ function Breadcrumb(props) {
   const dividerJSX = <BreadcrumbDivider icon={icon}>{divider}</BreadcrumbDivider>
   const sectionsJSX = []
 
-  sections.forEach(({ text, ...restSection }, index) => {
-    const key = `${text}-${index}`
```

Comment:
By applying the index to the key we are preventing React from reusing an unchanged component instance in the case that it moved positions. Index as a key is an antipattern, and while this sidesteps the core of that antipattern, it does so by negating the benefit provided in the reconciliation stage.

Text should be a sufficient key for bread crumbs (since each crumb should be unique, otherwise it's nonsensical), but, just in case, I adjusted it to allow for an optional `key` property should a user require this use case.